components/annot-vocabulary.vue

```diff
@@ -1,299 +1,356 @@
+                        break;
+                    
+                    case "row":
+                        instructionText = "Please provide a SNOMED-CT term";
+                        break;
+
+                    default:
+                        instructionText = "Please provide an appropriate vocabulary term";
+                        break;
+                }
+
+                return instructionText;
+            },            
+
+            placeholder() {
+
+                let placeHolderText = "";
+                switch ( this.options.mode ) {
+
+                    case "column":
+                        placeHolderText = "e.g. MoCA";
+                        break;
+
+                    case "row":
+                        placeHolderText = "e.g. Parkinson's Disease";
+                        break;
+
+                    default:
+                        break;
+                }
+
+                return placeHolderText;
+            },
+
+            saveButtonColor() {
+
+                // Bootstrap variant color of the button to save the annotation to the data table
+                return ( !this.saveButtonDisabled ) ? "success" : "secondary";
+            }
+        },
+
+        mounted() {
+            
+            // Initialize the mapping of all unique values as null
+            this.initializeMapping();
+        },        
+
+        methods: {
+
+            applyAnnotation() {
+
+                // If this is 'row' mode
+                if ( "row" === this.options.mode ) {
+
+                    // We want to use the annotated dataTable here in order to not overwrite previous
+                    // annotations from other components
+
+                    // 1. Create a local copy of the annotated table for transformation
+                    const transformedTable = structuredClone(this.dataTable.annotated);
+
+                    // 2. Transform all values in columns categorized as 'age' columns
+                    for ( let index = 0; index < transformedTable.length; index++ ) {
+                        for ( const columnName in transformedTable[index] ) {
+
+                                if ( this.relevantColumns.includes(columnName) ) {
+
+                                    // TODO: if "value" is a missing value or doesn't fit the heuristic, this will currently break!
+                                    transformedTable[index][columnName] = this.transformedValue(columnName, transformedTable[index][columnName]);
+                                }
+                        }
+                    }
+
+                    // 3. Trigger a save of this transformation to the annotated table in the store
+                    this.$emit("update:dataTable", {
+
+                        transformHeuristics: this.vocabularyMapping,
+                        transformedTable: transformedTable
+                    });
+                } 
+                // Else, this is 'column' mode
+                else {
```

Comment:
I agree that adding the comment here is helpful as a reminder that the only other option is "column" at the moment. But in this case I think we should make the code explicit and use `else if ("column" === ...)`